docs/source/projects/index.rst

```diff
@@ -0,0 +1,19 @@
+=====================
+Projects using PAVICS
+=====================
+
+Some Ouranos projects are using PAVICS as a mechanism to distribute results, code or documentation. Here you'll find links to project pages.
+
+.. toctree::
+   :maxdepth: 1
+
+   cruesdeca
+
+
+
+Project' related notebooks
+
+.. toctree::
+   :maxdepth: 2
+
+   notebooks/index
```

Comment:
I think  `docs/source/projects/notebooks/index.rst` is missing so we do not see the actual notebook on RtD https://pavics-sdi--200.org.readthedocs.build/en/200/projects/

Reply:
https://readthedocs.org/api/v2/build/12970031.txt

/home/docs/checkouts/readthedocs.org/user_builds/pavics-sdi/checkouts/200/docs/source/projects/index.rst:16: WARNING: toctree contains reference to nonexisting document 'projects/notebooks/index'

Since you are there, might as well fix this other missing file error:

/home/docs/checkouts/readthedocs.org/user_builds/pavics-sdi/checkouts/200/docs/source/notebooks/index.rst:11: WARNING: toctree contains reference to nonexisting document 'notebooks/finch'

Reply:
Argh, forgot to add the new directory to git.